rezervo/api/chain_config.py

```diff
@@ -50,13 +51,15 @@ def put_chain_user_creds(
     db_user = crud.user_from_token(db, settings, token)
     if db_user is None:
         raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED)
+    if not await get_chain(chain_identifier).verify_authentication(chain_user_creds):
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN)
     updated_config = crud.upsert_chain_user_creds(
         db, db_user.id, chain_identifier, chain_user_creds
     )
     if updated_config is None:
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND)
     background_tasks.add_task(refresh_cron, db_user.id, [chain_identifier])
-    return updated_config
+    return {"username": updated_config.username}
```

Comment:
This line currently throws an error in prod, that we just silently ignore. Reason being is that updated_config: ChainUser (username and password) is not compatible to the response_model: ChainUserProfile (only username)

Reply:
Just saw this error in the prod logs, nice catch! Could maybe return an actual ChainUserProfile model with the username from `updated_config`, just for clarity?

Reply:
Ah yes updated in https://github.com/mathiazom/rezervo/pull/35/commits/d38fc5749ebcc9e2aa977e790a78271f981a3247